src/main/python/jaccard.py

```diff
@@ -0,0 +1,46 @@
+class Jaccard():
+    """Class containing methods for computing Jaccard similarity between two sentences based on character n-gram overlap."""
+
+    @staticmethod
+    def text2ngrams(text, n=3):
+        """Convert text into character n-grams."""
+        return [text[i:i+n] for i in range(len(text)-n+1)]
+
+    @staticmethod
+    def jaccard_overlap(text1, text2):
+        """
+        Converts the text to character n-gram set and computes Jaccard overlap.
+        Calculates Jaccard overlap scores between two texts. 
+        """
+        set1 = set(Jaccard.text2ngrams(text1, n=3))
+        set2 = set(Jaccard.text2ngrams(text2, n=3))
+        intersection = set1.intersection(set2)
+        union = set1.union(set2)
+        return (len(intersection) / float(len(union)))
+
+    @staticmethod
+    def most_similar_passage(query, candidate_passages):
+        """Returns the passage with the highest Jaccard overlap with the query."""
```

Comment:
i didn't fix this because Python2 doesn't accept default argument plus this is placeholder model for now